### ISSUE TITLE
Very little is read by screenreaders

### ISSUE THREAD

**Issue by eschiebel** [TypeCellOS/BlockNote#591]
**Describe the bug**
Everything in the UI needs to be read by screenreaders, and it's not.

**To Reproduce**
- Turn on your screenreader.  (I'm using VoiceOver on chrome)
- on entering the editor it says "enter text or type for commands". it does not say "slash"
- type / to open the slash menu
- down arrow to navigate thru the menu
- nothing is announced.

Navigate around in the content and you get a mixture of the text entered and the slash menu prompt.

You can see this in the live editor on the https://www.blocknotejs.org/ home page

**Misc**
- Node version:
- Package manager:
- Browser: 
- [ ] I'm a [sponsor](https://github.com/sponsors/YousefED) and would appreciate if you could look into this sooner than later 💖

**Comment by eschiebel** [comment]
I can create more issues if you'd like, but there are other a11y issues in BlockNote wrt keyboard only users
- there are no focus indicators to let the keyboard user know where they are
- not all functionality is available to the keyboard user (e.g. the menu off the drag handle, or reordering blocks)
- the inline content formatting toolbar can't be navigated via the keyboard

**Comment by YousefED** [comment]
Thanks @eschiebel . We're planning an extensive accessibility sprint in April / May!

**Comment by YousefED** [comment]
Hi! We've made significant improvements in the latest release: https://github.com/TypeCellOS/BlockNote/pull/718

@eschiebel curious what you think about the update. Feel free to continuing the discussion in that PR!
